File: Examples/CSharp/Loading-and-Saving/ConvertSpecificPageToImage.cs
using System;
using System.IO;
using Aspose.Note;
using Aspose.Note.Saving;

namespace Aspose.Note.Examples.CSharp.Loading_Saving
{
    public class ConvertSpecificPageToImage
    {
        public static void Run()
        {
            ConvertDocumentToImage();
            SetOutputImageResolution();
        }

        public static void ConvertDocumentToImage()
        {
            // ExStart:ConvertSpecificPageToImage
            // ExFor:Document
            // ExFor:ImageSaveOptions
            // ExFor:SaveOptions.PageIndex
            // ExSummary:Shows how to save a document in png format.

            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_LoadingAndSaving();

            // Load the document into Aspose.Note.
            Document oneFile = new Document(dataDir + "Aspose.one");

            // Initialize ImageSaveOptions object 
            ImageSaveOptions opts = new ImageSaveOptions(SaveFormat.Png)
                                        {
                                            // Set page index
                                            PageIndex = 1
                                        };

            dataDir = dataDir + "ConvertSpecificPageToImage_out.png";

            // Save the document as PNG.
            oneFile.Save(dataDir, opts);

            // ExEnd:ConvertSpecificPageToImage

            Console.WriteLine("\nOneNote document converted successfully to image.\nFile saved at " + dataDir);
        }

        public static void SetOutputImageResolution()
        {
            // ExStart:SetOutputImageResolution
            // ExFor:Document
            // ExFor:ImageSaveOptions
            // ExFor:ImageSaveOptions.Resolution
            // ExFor:SaveFormat
            // ExSummary:Shows how to set a image resolution when saving document as image.

            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_LoadingAndSaving();

            // Load the document into Aspose.Note.
            Document doc = new Document(dataDir + "Aspose.one");

            dataDir = dataDir + "SetOutputImageResolution_out.jpg";

            // Save the document.
            doc.Save(dataDir, new ImageSaveOptions(SaveFormat.Jpeg) { Resolution = 220 });

            // ExEnd:SetOutputImageResolution
            
            Console.WriteLine("\nOneNote document output resolution setup successfully.\nFile saved at " + dataDir);
        }

        public static void SetOutputImageQuality()
        {
            // ExStart:SetOutputImageQuality
            // ExFor:Document
            // ExFor:ImageSaveOptions
            // ExFor:ImageSaveOptions.Quality
            // ExFor:SaveFormat
            // ExSummary:Shows how to set a image quality when saving document as image in JPEG format.

            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_LoadingAndSaving();

            // Load the document into Aspose.Note.
            Document doc = new Document(dataDir + "Aspose.one");

            dataDir = dataDir + "SetOutputImageResolution_out.jpg";

            // Save the document.
            doc.Save(dataDir, new ImageSaveOptions(SaveFormat.Jpeg) { Quality = 100 });
            
            // ExEnd:SetOutputImageQuality
            
            Console.WriteLine("\nOneNote document output quality setup successfully.\nFile saved at " + dataDir);
        }
    }
}

File: Examples/VisualBasic/Loading-and-Saving/ConvertSpecificPageToImage.vb
Imports System.IO
Imports Aspose.Note
Imports Aspose.Note.Saving
Namespace Loading_Saving
    Public Class ConvertSpecificPageToImage
        Public Shared Sub Run()
            ' ExStart:ConvertSpecificPageToImage
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_LoadingAndSaving()

            ' Load the document into Aspose.Note.
            Dim oneFile As New Document(dataDir & Convert.ToString("Aspose.one"))

            ' Initialize ImageSaveOptions object 
            Dim opts As New ImageSaveOptions(SaveFormat.Png)

            ' Set page index
            opts.PageIndex = 1

            dataDir = dataDir & Convert.ToString("ConvertSpecificPageToImage_out_.png")

            ' Save the document as PNG.
            oneFile.Save(dataDir, opts)
            ' ExEnd:ConvertSpecificPageToImage
            Console.WriteLine(Convert.ToString(vbLf & "OneNote document converted successfully to image." & vbLf & "File saved at ") & dataDir)
        End Sub
        Public Shared Sub SetOutputImageResolution()
            ' ExStart:SetOutputImageResolution
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_LoadingAndSaving()

            ' Load the document into Aspose.Note.
            Dim doc As New Document(dataDir & Convert.ToString("Aspose.one"))

            dataDir = dataDir & Convert.ToString("SetOutputImageResolution_out_.jpg")

            ' Save the document.
            doc.Save(dataDir, New ImageSaveOptions(SaveFormat.Jpeg) With {.Resolution = 220})
            ' ExEnd:SetOutputImageResolution
            Console.WriteLine(Convert.ToString(vbLf & "OneNote document output resolution setup successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace